apps/root/src/frame/components/navigation/index.tsx

```diff
@@ -137,10 +136,7 @@ const Navigation = ({
           ...DCA_ROUTE,
           label: intl.formatMessage(DCA_ROUTE.label),
           type: SectionType.link,
-          options: [
-            { ...DCA_CREATE_ROUTE, label: intl.formatMessage(DCA_CREATE_ROUTE.label), type: SectionType.link },
-            { ...DCA_POSITIONS_ROUTE, label: intl.formatMessage(DCA_POSITIONS_ROUTE.label), type: SectionType.link },
-          ],
+          activeKeys: [DCA_ROUTE.key, DCA_CREATE_ROUTE.key],
```

Comment:
Found one case here: If the user clicks on `Invest (DCA)`, the url would change into /positions every time.
This is annoying when you have been redirected to /create, and you click again on `Invest (DCA)`, the url changes but the site doesnt. 
I think we should also change the url of the navigation item if we have fetched this information in positionService